scripts/plant_life_hist_eu.py

```diff
@@ -68,7 +68,7 @@ def download(self, engine=None, debug=False):
             new_data.close()
             self.engine.auto_create_table(Table(key,
                                                 cleanup=Cleanup(correct_invalid_value,
-                                                                nulls=[-999.9])),filename=str("new" + key))
+                                                                missingValues=[-999.9])),filename=str("new" + key))
```

Comment:
I would try doing an install with at least one of them. Best case scenario I suspect that it's running, but the null values aren't getting replaced.